### ISSUE TITLE
Placeholders instead of raw data inside .from_tensor_slices

### ISSUE THREAD

**Issue by Uiuran** [sbrodehl/tf-boilerplate#9]
Hi, as i understood you output your training and testing batch slices to configure it from the loaded data, but i need to load data in a programed fashion, since it's a lot more data than your dataset. Then what iam trying to do is to feed a load of data as explained here:

https://www.tensorflow.org/guide/datasets#consuming_numpy_arrays.

However, i dont know if batch size, epochs of training and other configurations you have done will figure in the same way that its done in your code. Did you tried something like that for feeding data to your model ? 

note: iam asking to, maybe, help to expand it, i found your code very clean and well written, iam using it for reference since there is no good reference in training with session monitoring, even in TensorFlow Docs (may you send a contribution proposal to the TFlow docs with your code?).

Thank you in advance

**Comment by sbrodehl** [comment]
Datasets like [Fashion-MNIST](data/fashionmnist/__init__.py) or [MNIST](data/mnist/__init__.py) read numpy arrays and feed them to the tf.data.Dataset.
And as long as you return a tf.data.Dataset everything should work as expected.

**Comment by Uiuran** [comment]
Yes, that's my aim, however iam not playing to know do pop things with deep
learning, iam actually uncovering the whole lib (including low level) to do
serious research from previous-ly not worked data (such audio from
water-system). So iam trying to achieve these more general approaches, your
example is a pretty good one to start with but i need to see how to work
with the placeholder (for example, where i put the feed_dict of the
placeholder).

**Comment by sbrodehl** [comment]
The iterator of the dataset is initialized [here](tfbp.py#L149) (for the training part), so the feed dict of the initializer needs to be passed there.
Maybe it is a good idea to follow the [guide](https://www.tensorflow.org/guide/datasets#consuming_numpy_arrays) you posted.

**Comment by Uiuran** [comment]
Nice, thank you :-)
